src/Bootstrappers/LogChannelBootstrapper.php

```
@@ -0,0 +1,175 @@
+        }
+    }
+
+    protected function overrideChannelConfig(string $channel, array|Closure $override, Tenant $tenant): void
+    {
+        if (is_array($override)) {
+            // Map tenant attributes to channel config keys.
+            // If the tenant attribute is null,
+            // the override is ignored and the channel config key's value remains unchanged.
+            foreach ($override as $configKey => $tenantAttributeName) {
+                /** @var Tenant&Model $tenant */
+                $tenantAttribute = Arr::get($tenant, $tenantAttributeName);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Arr' in method 'overrideChannelConfig'. (undefined)

(StaticAccess)